internal/cmd/network/disable_protection_test.go

```diff
@@ -0,0 +1,42 @@
+package network_test
+
+import (
+	_ "embed"
```

Comment:
I see that the embed lib is sometimes imported but not used in the file, is that normal?

Reply:
That is required for `//go:embed` to work: https://pkg.go.dev/embed

Reply:
I understand, but we don't embed any file in this particular file.

Reply:
Removed all embed imports in files without actual embeds.